generate/generate.go

```diff
@@ -440,7 +440,7 @@ func (g *Generator) translatePkg(pkgPath string) error {
 
 // fileOptions will return the proto file options that have been set in the
 // gunk package. These include "JavaPackage", "Deprecated", "PhpNamespace", etc.
-func (g *Generator) fileOptions(pkg *loader.GunkPackage) (*desc.FileOptions, error) {
+func FileOptions(pkg *loader.GunkPackage) (*desc.FileOptions, error) {
```

Comment:
was this left in accidentally? doesn't seem to be needed outside of `generate`

Reply:
Yes, it was, thanks for catching it. I needed it to write tests